File: tuts/001-lightsail-gs/swift/Sources/GettingStartedScenario.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSLightsail

@main
struct GettingStartedScenario {
    static func main() async throws {
        let client = try LightsailClient(region: "us-east-1")
        let wrapper = LightsailWrapper(client: client)
        print("Running Lightsail getting started scenario...")
        // TODO: setup, interact, teardown
        let _ = wrapper
    }
}

File: tuts/002-vpc-gs/java/src/main/java/com/example/ec2/GettingStartedScenario.java
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.ec2;

import software.amazon.awssdk.services.ec2.Ec2Client;

public class GettingStartedScenario {
    public static void main(String[] args) {
        try (Ec2Client client = Ec2Client.builder().build()) {
            Ec2Wrapper wrapper = new Ec2Wrapper(client);
            System.out.println("Running Ec2 getting started scenario...");
            // TODO: setup, interact, teardown
            System.out.println("Scenario complete.");
        }
    }
}
